index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+  <meta charset="utf-8" />
+  <title>Personal Portfolio</title>
+  <meta name="viewport" content="width=device-width,initial-scale=1" />
+  <link rel="stylesheet" type="text/css" href="assets/css/style.css" />
+</head>
+
+<body>
+
+  <header>
+    <div class="wrapper menu">
+      <span class="menu-logo">My Logo</span>
+      <span class="menu-button"></span>
+    </div>
+  </header>
+
+  <main>
+
+    <section class="wrapper hero">
+      <h1 class="hero-title">Hey There. I’m Alex.</h1>
+      <div class="hero-subtitle">I'm a Software Developer</div>
+      <div class="hero-text">
+        I can help you build a product, feature or website Look through some of my work and experience!
+        If you like what you see and have a project you need coded, don’t hesitate to contact me.
+      </div>
```

Comment:
Great job on the UI! :grinning: I noticed that this text is currently wrapped in a `<div>` tag. To improve the semantic structure of your HTML, it would be more appropriate to use a `<p>` tag instead, as it represents a paragraph. :grinning:

:pray: Please consider making the necessary change and updating the wrapping element from `<div>` to `<p>` to ensure better semantic markup. This will help assistive technologies and search engines to better understand the content and improve overall accessibility.